specification/search/data-plane/Microsoft.Azure.Search.Data/preview/2017-11-11-preview/searchindex.json

```diff
@@ -224,10 +543,88 @@
           }
         }
       }
-      
     }
   },
   "definitions": {
+    "HitHighlights": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        }
+      },
+      "description": "Defines the generic document in an Azure Search index.",
+      "x-ms-external": true
+    },
+    "FacetResults": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        }
+      },
+      "x-ms-external": true
+    },
+    "DocumentSearchResult": {
+      "type": "object",
+        "properties": {
```

Comment:
nit: properties should be aligned with type

Reply:
Thanks. Fixed.